docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,230 @@
+
+	free(app_id);
+
+	return;
+   }
+   ```
+
+<a name="resourceusage_get"></a>
+## Getting Battery Usage Information of Particular Resource
+
+The battery usage information contains combination of the resource ID and the time duration.
+
+1. Fetching the battery usage information of a particular resource over a certain duration of time.
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, as shown in the following code:
+
+   ```
+   #include <battery_monitor.h>
+
+   void
+   func(void)
+   {
+	int error_code = BATTERY_MONITOR_ERROR_NONE;
+	int battery_usage = -1;
+	battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_DISPLAY;
+	battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+
+	error_code = battery_monitor_get_total_usage_by_resource_id(resource_id, duration_val, &battery_usage);
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("Battery Usage Display in last 24 hrs is [%d]", battery_usage);
+
+	resource_id = BATTERY_MONITOR_RESOURCE_ID_WIFI;
+	error_code = battery_monitor_get_total_usage_by_resource_id(resource_id, duration_val, &battery_usage);
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("Battery Usage Wifi in last 24 hrs is [%d]", battery_usage);
+
+	/* Similarly for other resources...
+	.
+	.
+	.
+	*/
+
+	return;
+   }
+   ```
+<a name="resource_key"></a>
+## Resource Keys
+
+The following table lists the available resource keys, which are part of `battery_monitor_resource_id_e` enumeration:
+
+**Table: Resource Keys**
+ | Key                                            | Description                                   |
+ |------------------------------------------------|-----------------------------------------------|
+ | `BATTERY_MONITOR_RESOURCE_ID_BLE`              | Indicates resource key for Bluetooth.         |
+ | `BATTERY_MONITOR_RESOURCE_ID_WIFI`             | Indicates resource key for Wifi.              |
+ | `BATTERY_MONITOR_RESOURCE_ID_CPU`              | Indicates resource key for CPU.               |
+ | `BATTERY_MONITOR_RESOURCE_ID_DISPLAY`          | Indicates resource key for Display.           |
+ | `BATTERY_MONITOR_RESOURCE_ID_DEVICE_NETWORK`   | Indicates resource key for Device Network.    |
+ | `BATTERY_MONITOR_RESOURCE_ID_GPS_SENSOR`       | Indicates resource key for GPS Sensor.        |
+ | `BATTERY_MONITOR_RESOURCE_ID_HRM_SENSOR`       | Indicates resource key for HRM Sensor.        |
+
+<a name="duration_key"></a>
+## Duration Types
+
+The following table lists the available duration type keys, which are part of `battery_monitor_duration_type_e` enumeration:
+
+**Table: Duration Type Keys**
+ | Key                                            | Description                                          |
+ |------------------------------------------------|------------------------------------------------------|
```

Comment:
From line 225 to line 226.
How about removing "Indicates" and "i.e."?

Duration of last one day (24 hrs)
Duration of last one week (7 days)